core/ir/opnd_api.h

```diff
@@ -42,11 +42,6 @@
  * @brief Functions and defines to create and manipulate instruction operands.
  */
 
-/* Catch conflicts if ucontext.h is included before us */
-#if defined(DR_REG_ENUM_COMPATIBILITY) && (defined(REG_EAX) || defined(REG_RAX))
-#    error REG_ enum conflict between DR and ucontext.h!  Use DR_REG_ constants instead.
```

Comment:
This is a useful check: I don't think it should just be thrown away.  If you hit it, that means there is a real enum conflict, doesn't it?

Reply:
I don't know. It happened on the CI but I couldn't reproduce it locally. If I'm lucky the problem will go away when I stop using C++.

Reply:
It did go away.